build-logic/performance-testing/src/main/kotlin/gradlebuild/performance/PerformanceTestPlugin.kt

```diff
@@ -251,6 +255,17 @@ class PerformanceTestPlugin : Plugin<Project> {
         }
     }
 
+    private
+    fun Project.configureAndroidStudioInstallationPlugin() {
+        val isRunningOnTeamCity = System.getenv("TEAMCITY_VERSION") != null
+        plugins.apply("gradlebuild.android-studio-test-setup")
```

Comment:
I think this should be better in the project where we setup the templates `gradlebuild.performance-templates`, instead of here. Or the code which adds a template here should have a configuration whether the test project is an Android project or not.